src/webapp/cilogon_ldap.py

```diff
@@ -0,0 +1,44 @@
+import ldap3
+
+
+def get_contact_cilogon_id_map(global_data):
+    """ return contacts dict, limited to users with a CILogonID """
+    contacts = global_data.get_contacts_data().users_by_id;
+    return { k: v for k, v in contacts.items() if v.cilogon_id is not None }
```

Comment:
I think this is a function that we want to drop in the future, i.e. we want to grab contact info directly from CILogon and drop the contact db

Reply:
Sure and come to think of it maybe it doesn't really belong in the new cilogon_ldap module anyway. It just returns a subset of the global contacts_data users_by_id map, for users with a cilogon_id.

And yeah it will have to go away when the contact db does.

Reply:
I think it's fine to move it here for now